packages/eslint-config-base/package.json

```diff
@@ -23,6 +23,10 @@
     "madebywild"
   ],
   "main": "index.js",
+  "dependencies": {
+    "eslint-plugin-prettier": "^3.4.0",
+    "eslint-config-prettier": "^8.3.0"
```

Comment:
Since they are used from all the configurations, we can flag them as the `peerDependencies`.